### PR TITLE
Admin manager categories

### PR DESCRIPTION
**Pull request by NguyenThiHangB**

### CODE REVIEW

**Comment by NguyenThiHangB** [comment]
ready

**Comment by huongnguyenmta** [comment]
commented
pull có 2 commit nhé (facepalm5)
đính kèm link task redmine vào pr luôn đi em -_-

**Comment by NguyenThiHangB** [comment]
ready
Link task: https://dev.framgia.com/issues/185252

**Comment by NguyenThiHangB** [comment]
ready
Link task: https://dev.framgia.com/issues/185252

File: app/views/categories/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<%= form_for @category do |f| %>
+<%= render "shared/error_messages", object: @category %>
```
Comment (mdmahmudurrahman) [inline app/views/categories/_form.html.erb]: Please add indentation as this tag is inside another block

File: app/views/categories/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  <div class="form-group">
+    <%= f.label :title %>
+    <%= f.text_field :title, class: "form-control" %>
+  </div>
+  <div class="form-group">
+    <%= f.label :description %>
+    <%= f.text_area :description, class: "form-control", placeholder: t(".description"), :rows => Settings.category.form.rows %>
```
Comment (mdmahmudurrahman) [inline app/views/categories/_form.html.erb]: 1. Hash racket syntax is deprecated. Please use symbol instead. 
`rows: Settings.category.form.rows`
2. Break this long line

**Comment by mdmahmudurrahman** [comment]
commented

**Comment by NguyenThiHangB** [comment]
ready

**Comment by mdmahmudurrahman** [comment]
commented
This (pr) can be (merged)
CC: CC: [To:1818111] Nguyen Thi Huong (94)

**Pull request merged**